pipelines/_ingredients/SingleFrame.yaml

```diff
@@ -0,0 +1,19 @@
+description: Single-frame pipeline
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipe.yaml
+    include:
+      - processCcd
+      - initialPviCore
+      - getRegionTimeFromVisit
+      - mpSkyEphemerisQuery
+tasks:
+  ssSingleFrameAssociation:
+    class: lsst.ap.association.SsSingleFrameAssociationTask 
+subsets:
+  processCcd:
+    subset:
+      - isr
+      - calibrateImage
+      - ssSingleFrameAssociation
+    description: >
+      An alias of SingleFrame to use in higher-level pipelines.
```

Comment:
Please edit this description because calling processCcd an alias of singleFrame is quite confusing.